### PR TITLE
Integration tests: cleanups, more mounts, drop PluginsTest

### PR DESCRIPTION
**Pull request by tehasdf**
Dropping the unused cruft, mostly

### CODE REVIEW

File: tests/integration_tests/conftest.py [hunk 1]
```diff
@@ -45,7 +45,9 @@ def pytest_addoption(parser):
 # pairs of (source path, [list of target virtualenvs])
 # TODO fill this in as needed, when needed
 sources = [
+    ('cloudify-cli/cloudify_cli', ['/opt/cfy']),
     ('cloudify-common/cloudify', ['/opt/manager/env', '/opt/mgmtworker/env']),
```
Comment (geokala) [inline tests/integration_tests/conftest.py:49]: I know it's not part of this change, but isn't this needed in /opt/manager/env too?

Comment (tehasdf) [reply tests/integration_tests/conftest.py:49]: nonono, thankfully we don't install the cli into the restservice venv

Comment (geokala) [reply tests/integration_tests/conftest.py:49]: I mean cloudify-common

Comment (geokala) [reply tests/integration_tests/conftest.py:49]: Ignore me, I misread this line.

**Pull request merged**